src/lib/utils.ts

```
@@ -1,6 +1,6 @@
-import { ClassValue, clsx } from 'clsx';
-import { twMerge } from 'tailwind-merge';
+import { clsx, type ClassValue } from "clsx"
```

ESLint

[error] Replace `"clsx"` with `'clsx';`

(prettier/prettier)